README.md

```diff
@@ -148,12 +140,15 @@ This is an example of how to list things you need to use the software and how to
    ```sh
    postgres=# exit
    ```
-5. install requirements:
+4. install requirements:
    ```sh
    pip install -r requirements/requirements-dev.txt
    ```
-
-6. Copy `.env.template.local` file, rename to `.env` and use variables for your local postgres database.
+   set up the git hook scripts
+   ```sh
```

Comment:
I left a description of the instructions for clarity, is it better to remove them?

Reply:
Nope that works. Thank you